apps/product/tasks.py

```
@@ -1,207 +1,983 @@
+            if product.exists():
+                wildberries_product = product.filter(marketplace_type="wildberries")
+                if wildberries_product.exists():
+                    W_product = wildberries_product.first()
+                else:
+                    W_product = False
+                
+                ozon_product = product.filter(marketplace_type='ozon')
+                if ozon_product.exists():
+                        ozon_product = ozon_product.first()
+                else:
+                    ozon_product = False
```

Ruff

Use ternary operator `ozon_product = ozon_product.first() if ozon_product.exists() else False` instead of `if`-`else`-block

Replace `if`-`else`-block with `ozon_product = ozon_product.first() if ozon_product.exists() else False`

(SIM108)